src/dbnode/storage/index/block.go

```diff
@@ -253,30 +257,74 @@ func (b *block) Query(
 		return false, err
 	}
 
-	var (
-		size       = results.Size()
-		brokeEarly = false
-	)
+	size := results.Size()
```

Comment:
wanna add the var back? I think it was cleaner. I assume you removed it unintentionally while refactoring